README.md

```diff
@@ -45,7 +44,7 @@ Goals:
     anyone, such as [in-toto] or [Binary Authorization].
 -   Make it possible to write automated policies that take advantage of
     structured information.
--   Fit within the [SLSA Framework][SLSA]. The [provenance] format defined here
+-   Fit within the [SLSA Framework][SLSA].
     is the official SLSA recommendation.
```

Comment:
I think something got mangled here.  Now it just reads "Fit within the [SLSA Framework][SLSA].
    is the official SLSA recommendation."

Reply:
Oops, fixed, thanks for catching.